### PR TITLE
[REF] Create Alter Column function in upgrade base class

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This creates an alterColumn function similar to that of addColumn and refactors the upgrade code from #25143 to use the new method

Before
----------------------------------------
No standard method for modifying columns in the database during upgrades

After
----------------------------------------
One standard method

ping @colemanw

### CODE REVIEW

File: CRM/Upgrade/Incremental/Base.php [hunk 1]
```diff
@@ -585,4 +585,26 @@ public static function rebuildMultilingalSchema($ctx, $version = NULL) {
     return TRUE;
   }
 
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE): bool {
+    $locales = CRM_Core_I18n::getMultilingual();
+    $queries = [];
+    if ($localizable) {
+      if ($locales) {
+        foreach ($locales as $locale) {
+          $queries[] = "ALTER TABLE `$table` CHANGE `{$column}_{$locale}` `description_{$locale}` $properties";
```
Comment (colemanw) [inline CRM/Upgrade/Incremental/Base.php:594]:
```suggestion
          $queries[] = "ALTER TABLE `$table` CHANGE `{$column}_{$locale}` `{$column}_{$locale}` $properties";
```

File: CRM/Upgrade/Incremental/Base.php [hunk 2]
```diff
@@ -585,4 +585,26 @@ public static function rebuildMultilingalSchema($ctx, $version = NULL) {
     return TRUE;
   }
 
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE): bool {
+    $locales = CRM_Core_I18n::getMultilingual();
+    $queries = [];
+    if ($localizable) {
+      if ($locales) {
```
Comment (colemanw) [inline CRM/Upgrade/Incremental/Base.php:592]: These two if statements could be combined to 
```suggestion
    if ($localizable && $locales) {
```

**Comment by seamuslee001** [comment]
thanks @colemanw I think I have patched up both things now

**Comment by seamuslee001** [comment]
thanks @demeritcowboy @colemanw

**Pull request merged**